TYPO3.Flow/Classes/TYPO3/Flow/Http/RequestHandler.php

```diff
@@ -129,7 +134,7 @@ public function handleRequest()
      */
     public function getHttpRequest()
     {
-        return $this->request;
+        return $this->componentContext->getHttpRequest();
```

Comment:
This only works if `handleRequest()` has been invoked before.. Why not check for null first and prevent the Fatal Error that occurs otherwise?